src/sentry/static/sentry/app/routes.jsx

```diff
@@ -1198,20 +1198,102 @@ function routes() {
+                  )
+                }
+                component={errorHandler(LazyLoad)}
+              />
+              <Route
+                path="events/"
+                componentPromise={() =>
+                  import(
+                    /* webpackChunkName: "GroupEvents" */ 'app/views/organizationGroupDetails/groupEvents'
+                  )
+                }
+                component={errorHandler(LazyLoad)}
+              />
+              <Route
+                path="similar/"
+                componentPromise={() =>
+                  import(
+                    /* webpackChunkName: "GroupSimilarIssues" */ 'app/views/organizationGroupDetails/groupSimilarIssues'
+                  )
+                }
+                component={errorHandler(LazyLoad)}
+              />
+              <Route
+                path="tags/"
+                componentPromise={() =>
+                  import(
+                    /* webpackChunkName: "GroupTags" */ 'app/views/organizationGroupDetails/groupTags'
+                  )
+                }
+                component={errorHandler(LazyLoad)}
+              />
+              <Route
+                path="tags/:tagKey/"
+                componentPromise={() =>
+                  import(
+                    /* webpackChunkName: "GroupTagsValues" */ 'app/views/organizationGroupDetails/groupTagValues'
+                  )
+                }
+                component={errorHandler(LazyLoad)}
+              />
+              <Route
+                path="feedback/"
+                componentPromise={() =>
+                  import(
+                    /* webpackChunkName: "GroupUserFeedback" */ 'app/views/organizationGroupDetails/groupUserFeedback'
+                  )
+                }
+                component={errorHandler(LazyLoad)}
+              />
+              <Route
+                path="attachments/"
+                componentPromise={() =>
+                  import(
+                    /* webpackChunkName: "GroupEventAttachments" */ 'app/views/organizationGroupDetails/groupEventAttachments'
+                  )
+                }
+                component={errorHandler(LazyLoad)}
+              />
+              <Route
+                path="similar/"
+                componentPromise={() =>
+                  import(
+                    /* webpackChunkName: "GroupSimilarIssues" */ 'app/views/organizationGroupDetails/groupSimilarIssues'
+                  )
+                }
+                component={errorHandler(LazyLoad)}
+              />
+              <Route
+                path="merged/"
+                componentPromise={() =>
+                  import(
+                    /* webpackChunkName: "GroupMerged" */ 'app/views/organizationGroupDetails/groupMerged'
+                  )
+                }
+                component={errorHandler(LazyLoad)}
+              />
+            </Route>
```

Comment:
Do these sub routes make sense when we are only targeting a single event? What does it mean to look at the Merged, or Similar Issues tab when we have a single event?

Additionally will you have to update all of these views to only support one event?

Reply:
I had to add new routes because of 2 problems that I noticed:

1 - If you wanna see a specific event of an issue, the URL usually contains` /events/:eventId/` but by clicking on some different tab than details, the URL gets updated removing ` /events/:eventId/, consequently fetching a different event.

> I truly believe that this is wrong because as I user still wanna see data from that specific event.

2 - I really need the correct trace ID to provide smooth navigation between "Similar issues by tracking ID" to the user. I am currently passing down the trace ID of the fetched event response made by groupDetails (HOC) to the `Similar Issues Tab`. 

The data used by the fetch function currently comes from the URL. This means the following problem may happen:

1 - User is on the similar issues tab
2 - User clicks on some event that contains the trace ID X.
3 - URL will get updated bringing the user to the details page that contains the data of that specific event X.
4 - User clicks again on the tab Similar issues
5 - URl gets updated, removing the eventId from the URL
6 - groupDetails will fetch a new event with a different trace ID - Y
7 - The list rendered on the Similar Issues tab will no longer contain related issues by the trace ID X because the fetched event has the trace ID Y.

-----------------------------------------------------------------------------------------------------------------

We are currently rendering the tabs according to the URL and I preferred to add new routes instead of changing this structure now (It's an important refactor). Maybe in the future, we can update it by passing for example only a parameter to the URL. `/issues/:groupId/?tab=details`

Reply:
> Additionally will you have to update all of these views to only support one event?

I don't think that all the tabs will need to be updated, but ..some yes..like tabs. 

I already have done that by updating the baseUrl.

Reply:
Hmm, what about the `Events` tab?

Reply:
the links of the `Events` tab are correct 😉

Reply:
Will it be unexpected for the user to see a list of all events when there is an event ID in the URL?

Reply:
Yes I agree with...maybe we could rename the tab to "Related Events" when there is an event id in the URL...Please what do you think?